examples/binary/README.md

```diff
@@ -2,7 +2,10 @@
 
 [![](https://github.com/artipie/artipie/workflows/Proof::binary/badge.svg)](./examples/binary)
 
-Try this `repo.yaml` file:
+This directory contains a basic example of how to can use artipie as a storage for binary files.
```

Comment:
@Sammers21 
part "how to can use artipie as a storage for binary files" is still not correct, I'd suggest "how artipie can be used as ... "